### DISCUSSION TITLE
Sittings explained

### DISCUSSION THREAD

**Discussion by HassanDalwi** [ReVanced/revanced-discussions#895] [General]
I wish someone make a video and explain revanced settings. So we can understand every single settings. 

For example every settings need to be show exactly what happened if we turn on this. And go to YouTube to see what happenes

**Comment by kazimmt** [comment]
Untill someone make a video

You can see features from here
https://telegra.ph/ReVanced-Features-11-08
(Screenshot included)

**Reply by HassanDalwi** [reply]
Good

**Reply by LisoUseInAIKyrios** [reply]
https://www.reddit.com/r/revancedapp/s/ddsC65O5wu

**Reply by kazimmt** [reply]
@LisoUseInAIKyrios 
I'm preparing this
I do have most screenshots. But need your help to find others.
You can knock me in Telegram @kaziMMT
![image](https://github.com/ReVanced/revanced-discussions/assets/82371061/d99da9cd-dd61-4e36-8e54-e7e5fe2544a8)

**Reply by LisoUseInAIKyrios** [reply]
I don't use Telegram or Discord.  Sorry.

More discussion on adding documentation for end users is here:
https://github.com/ReVanced/revanced-patches/issues/2618

**Reply by kazimmt** [reply]
@LisoUseInAIKyrios I'm developing this
https://kazimmt.github.io/rv-features/youtube/

I do have screenshots for most of the features. Only few features are missing.

**Reply by kazimmt** [reply]
It's completed. I've added screenshots for almost all features.
